WatchDogIntelliJPlugin/WatchDog/WatchDog.iml

```diff
@@ -19,10 +19,11 @@
         <SOURCES />
```

Comment:
Can you unstage these changes? (Or is there any specific reason for changing the order?)